project/LibraryDependencies.scala

```diff
@@ -2,31 +2,17 @@ import sbt._
 
 object LibraryDependencies {
 
-  private val play25Version = "2.5.19"
-  private val play26Version = "2.6.23" // this version is pinned, since later versions are incompatible with the version of akka that reactivemongo was build with on scala 2.11.
-  private val play27Version = "2.7.4"
+  private val play26Version = "2.6.25"
```

Comment:
version of Play 2.6 doesn't need pinning back now that we don't build for Scala 2.11